Functional/#4767-change-fucntionality-with-deactivating-tariff

JJJazl opened the pull request:
1) changed describe in method deactivateTariff of SuperAdminController
2) changed method deactivateTariffCard in SuperAdminServiceImpl. Only deactivating is available now
3) removed method deleteTariff int SuperAdminServiceImplTest

renat-safarov committed:
1) changed describe in method deactivateTariff of SuperAdminController
2) changed method deactivateTariffCard in SuperAdminServiceImpl. Only deactivating is available now.
3) removed method deleteTariff int SuperAdminServiceImplTest

OlegVat commented on service/src/main/java/greencity/service/ubs/SuperAdminServiceImpl.java:
```diff
@@ -41,6 +41,7 @@
 import javax.persistence.EntityNotFoundException;
 import java.time.LocalDate;
 import java.util.*;
+import java.util.function.Function;
```
remove redundant import to get rid of code smell

RomaMocherniuk commented on service/src/main/java/greencity/service/ubs/SuperAdminServiceImpl.java:
```diff
@@ -630,16 +631,30 @@ public void setTariffLimitBySumOfOrder(Long tariffId, EditPriceOfOrder dto) {
+
+        tariffsInfoRepository.save(tariffsInfo);
+
+        return "Deactivated";
```
Do we need it?
Now this method is responsible for deactivating only

RomaMocherniuk commented on service/src/main/java/greencity/service/ubs/SuperAdminServiceImpl.java:
```diff
@@ -630,16 +631,30 @@ public void setTariffLimitBySumOfOrder(Long tariffId, EditPriceOfOrder dto) {
+    @Transactional
     public String deactivateTariffCard(Long tariffId) {
         TariffsInfo tariffsInfo = tryToFindTariffById(tariffId);
-        if (tariffsInfo.getOrders().isEmpty()) {
```
please add a test that we also deactivate tariff if `tariffsInfo.getOrders().isEmpty()`

renat-safarov committed:
1) changed response type in method deactivateTariff() of SuperAdminController class
2) changed return type in method deactivateTariffCar() of SuperAdminService interface and SuperAdminServiceImpl class
3) changed test method deactivateTariff() in SuperAdminServiceImplTest and added new method deactivateTariffTestThrows()
5) removed extra imports in SuperAdminServiceImplTest and SuperAdminServiceImpl
